### PR TITLE
feat: initial commit

### PR DESCRIPTION
**Pull request by kellyjosephprice**
[![PR App][icn]][demo] | Fix RM-6810
:-------------------:|:----------:

## 🧰 Changes

A fresh start.

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,34 +14,30 @@ By default, the updated markdown package exports a function which takes a string
 
 ```jsx
 import React from 'react';
-import rdmd from "@readme/markdown";
+import rdmdx from '@readme/mdx';
```
Comment (erunion) [inline README.md:17]: We calling this package `mdx` or `rdmdx`?

File: .eslintrc [hunk 2]
```diff
@@ -1,8 +1,5 @@
 {
-  "extends": [
-    "@readme/eslint-config",
-    "@readme/eslint-config/react"
-  ],
+  "extends": ["@readme/eslint-config", "@readme/eslint-config/react"],
```
Comment (erunion) [inline .eslintrc]:
```suggestion
  "extends": ["@readme/eslint-config", "@readme/eslint-config/react", "@readme/eslint-config/typescript"],
```

**Review by erunion — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kellyjosephprice) [reply README.md:17]: I always thought of `RDMD` as an acronym of `@readme/markdown`, and so `rdmdx` would be the shortened form of `@readme/mdx`?

Comment (erunion) [reply README.md:17]: works for me

**Comment by kellyjosephprice** [comment]
I'm going to merge this, but I welcome any other feedback.

**Pull request merged**